src/test/scala/is/hail/io/LoadBgenSuite.scala

```diff
@@ -210,6 +212,43 @@ class LoadBgenSuite extends SparkSuite {
     packedInput
   }
 
+  @Test def testDosage() {
+    val sc = hc.sc
+    hc.indexBgen("src/test/resources/example.8bits.bgen")
+    val vds = hc.importBgen("src/test/resources/example.8bits.bgen"
+      // , nPartitions = Some(8)
+      , nPartitions = Some(1)
+    )
+
+    val mask = Array.fill(vds.nSamples)(true)
+    val incomplete = Array(13, 172, 273, 319, 441)
```

Comment:
use `incompleteSampleIndices` to be consistent with `completeSampleIndices` below